index.html

```diff
@@ -8,5 +8,7 @@
 </head>
 <body>
   <h1>Hello World!</h1>
+  <iframe width="560" height="315" src="https://www.youtube.com/embed/QGjzs7-t3uU" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 </body>
+<a rel="license" href="http://creativecommons.org/licenses/by-nc-sa/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-nc-sa/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-nc-sa/4.0/">Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International License</a>.
```

Comment:
If you would like to include this link, it should be moved to appear before `</body>`